Description: Do not invent our own users, use "proxy" and "nobody".
Author: Adrian Bridgett <bridgett@debian.org>
Forwarded: not-needed
Last-Update: 2010-07-02

--- a/example/sockd.conf
+++ b/example/sockd.conf
@@ -76,17 +76,18 @@
 #
 
 # when doing something that can require privilege, it will use the
-# userid "sockd".
-#user.privileged: sockd
+# userid:
+user.privileged: proxy
 
-# when running as usual, it will use the unprivileged userid of "sockd".
-#user.unprivileged: sockd
+# when running as usual, it will use the unprivileged userid of:
+user.unprivileged: nobody
 
 # If you are not using libwrap, no need for the below line, so leave
 # it commented.
 # If you compiled with libwrap support, what userid should it use
 # when executing your libwrap commands?  "libwrap".
 #user.libwrap: libwrap
+user.libwrap: nobody
 
 
 #
